j5/backends/hardware/sr/v4/power_board.py

```diff
@@ -116,7 +116,7 @@ def set_power_output_enabled(
             cmd = CMD_WRITE_OUTPUT[identifier]
         except KeyError:
             raise ValueError(f"Invalid power output identifier {identifier!r}; "
```

Comment:
Line 119 is the culprit that's failing the tests. I propose putting the string on a new line, although that will reduce consistency.
```suggestion
            raise ValueError(
                f"Invalid power output identifier {identifier!r}; "
                f"valid identifiers are {CMD_WRITE_OUTPUT.keys()}."
            ) from None
```

Reply:
There's also a missing full stop on line 109, but adding it will push the line over 90 characters.

Reply:
> although that will reduce consistency.

We don't mind as long as it passes the linter :)